server/src/main/java/io/crate/expression/reference/sys/snapshot/SysSnapshots.java

```diff
@@ -123,6 +117,22 @@ private static SysSnapshot toSysSnapshot(Repository repository,
         );
     }
 
+    private static SysSnapshot toSysSnapshot(String repositoryName,
+                                             SnapshotsInProgress.Entry entry,
+                                             List<String> partedTables) {
+        return new SysSnapshot(
+            entry.snapshot().getSnapshotId().getName(),
+            repositoryName,
+            entry.indices().stream().map(IndexId::getName).toList(),
+            partedTables,
+            entry.startTime(),
+            0L,
+            Version.CURRENT.toString(),
+            SnapshotState.IN_PROGRESS.name(),
+            Collections.emptyList()
+        );
+    }
```

Comment:
Values are taken from `SnapshotInfo(entry)` ctor. 

`SnapshotInfo(entry)` will be removed later BTW.